### DISCUSSION TITLE
tool flags

### DISCUSSION THREAD

**Discussion by beryl-factset** [actions/virtual-environments#3571] [Ideas]
Has there been a discussion about making flags for tools?
right now, the only way to change if tool x (Rust for example)  is installed is to manually go in  to each json file (windows2016.json, windows2019.json) and remove it.  That also leaves things  trying to run like in the Software Reports. 

1)  can we make everything get called from one spot? 
2) set flags to determine what software gets installed/omitted?

**Comment by miketimofeev** [comment]
@beryl-factset Hi! We've thought about that approach and there is a chance that we give it a try one day, but not in the nearest future.
